pennylane/devices/default_tensor.py

```diff
@@ -156,15 +160,43 @@ class DefaultTensor(Device):
             statistics like expectation values and variances is performed analytically.
         method (str): Supported method. Currently, only ``"mps"`` is supported.
         dtype (type): Datatype for the tensor representation. Must be one of ``np.complex64`` or ``np.complex128``.
-            Default is ``np.complex128``.
-        **kwargs: keyword arguments. The following options are currently supported:
-
-            ``max_bond_dim`` (int): Maximum bond dimension for the MPS simulator.
-                It corresponds to the number of Schmidt coefficients retained at the end of the SVD algorithm when applying gates. Default is ``None``.
-            ``cutoff`` (float): Truncation threshold for the Schmidt coefficients in a MPS simulator. Default is ``np.finfo(dtype).eps``.
-            ``contract`` (str): The contraction method for applying gates. It can be either ``auto-mps`` or ``nonlocal``.
-                ``nonlocal`` turns each gate into a MPO and applies it directly to the MPS, while ``auto-mps`` swaps nonlocal qubits in 2-qubit gates to be next
-                    to each other before applying the gate, then swaps them back. Default is ``auto-mps``.
+        **kwargs: keyword arguments for the device, passed to the ``quimb`` backend.
+
+    Keyword Args:
+        max_bond_dim (int): Maximum bond dimension for the MPS method.
+            It corresponds to the number of Schmidt coefficients retained at the end of the SVD algorithm when applying gates. Default is ``None``.
+        cutoff (float): Truncation threshold for the Schmidt coefficients in the MPS method. Default is ``np.finfo(dtype).eps``.
+        contract (str): The contraction method for applying gates in the MPS method. It can be either ``auto-mps`` or ``nonlocal``.
+            ``nonlocal`` turns each gate into a MPO and applies it directly to the MPS, while ``auto-mps`` swaps nonlocal qubits in 2-qubit gates to be next
+            to each other before applying the gate, then swaps them back. Default is ``auto-mps``.
+
+    **Example:**
+
+    The following code demonstrates how to create a very simple quantum circuit using the ``default.tensor`` device:
+
+    .. code-block:: python
+
+        import pennylane as qml
+        import numpy as np
+
+        num_qubits = 80
+
+        dev = qml.device("default.tensor", wires=num_qubits, method="mps", max_bond_dim=100)
```

Comment:
More for my understanding, what's a typical size for `max_bond_dim`?

Reply:
I asked myself the same question. Although it is not properly my field, I think the minimum `max_bond_dim` to reach a good approximation depends on several factors, but mostly the size of the circuit and the level of the entanglement in the circuit itself. Generally speaking, the values I have seen range from about 50 to 500.

This is just a minimal usage example, although in this case the `max_bond_dim` does not play a role so we can remove it. My idea was to ask people in the field to provide some usage examples that we can insert in an expandable section in the doc:
```
.. details::
            :title: Usage Details
```
with complicated use cases. Please let me know what you think